Makefile

```diff
@@ -132,3 +136,31 @@ vet:
 # Run tests
 test-unit: generate fmt vet manifests
 	go test ./... -coverprofile cover.out
+
+##################################################
+# Bundle                                           #
+##################################################
+
+# Default bundle image tag
+BUNDLE_IMG ?= controller-bundle:$(VERSION)
+# Options for 'bundle-build'
+ifneq ($(origin CHANNELS), undefined)
+BUNDLE_CHANNELS := --channels=$(CHANNELS)
+endif
+ifneq ($(origin DEFAULT_CHANNEL), undefined)
+BUNDLE_DEFAULT_CHANNEL := --default-channel=$(DEFAULT_CHANNEL)
+endif
+BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
+
+# Generate bundle manifests and metadata, then validate generated files.
+.PHONY: bundle
+bundle: manifests
```

Comment:
If I am not mistaken, the `make bundle` asks for some values, is there a way how to prefill them?